lib/mintacoin/accounts/crypto/stellar.ex

```diff
@@ -4,47 +4,101 @@ defmodule Mintacoin.Accounts.Stellar do
   """
 
   alias Mintacoin.Accounts.Crypto.AccountResponse
-  alias Stellar.{Horizon, Horizon.Transaction, KeyPair, TxBuild}
 
-  @type status :: :ok | :error
-  @type stellar_response :: map()
+  alias Stellar.{
+    Horizon,
+    Horizon.Transaction,
+    KeyPair,
+    TxBuild,
+    TxBuild.CreateAccount,
+    TxBuild.SequenceNumber,
+    TxBuild.Signature
+  }
+
+  @type create_account :: CreateAccount.t()
+  @type error :: {:error, any()}
+  @type funder_account_information :: map()
+  @type key_pair :: {public_key(), secret_key()}
   @type public_key :: String.t()
   @type secret_key :: String.t()
-  @type error :: {:error, any()}
+  @type sequence_number :: SequenceNumber.t()
+  @type signature :: Signature.t()
+  @type status :: :ok | :error
+  @type stellar_response :: map()
+  @type tx_envelope :: binary()
```

Comment:
@Odraxs update this in this way
```suggestion
    @type tx_envelope :: {:ok, envelope()} | {:error, atom()}
```

and the `envelop()` type should be `String.t()`